doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1056,6 +1071,7 @@ I/O
 - Bug in :meth:`DataFrame.to_sql` when reading DataFrames with ``-np.inf`` entries with MySQL now has a more explicit ``ValueError`` (:issue:`34431`)
 - Bug in :meth:`read_excel` that was raising a ``TypeError`` when ``header=None`` and ``index_col`` given as list (:issue:`31783`)
 - Bug in "meth"`read_excel` where datetime values are used in the header in a `MultiIndex` (:issue:`34748`)
+- Bug in :meth:`get_handle` was passing ``encoding`` to ``open`` when in binary mode (:issue:`35058`)
```

Comment:
can u reference only use facing functions here (to_csv)

Reply:
I tried to address that